### PR TITLE
Prepare release of version 4.1.0

### PR DESCRIPTION
**Pull request by susnux**
Would be great to have the Typescript fixes released, so I prepared the changelog and bumped the package version.

If accepted can someone with admin rights spin the npm release?

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+## v4.1.0 (2023-06-23)
+
+### :bug: Fixed bugs
+
+- Make sure Typescript projects with `node16` module resolution can import the package [\#845](https://github.com/nextcloud/nextcloud-dialogs/pull/845) ([@susnux](https://github.com/susnux))
+- Ensure all package dependencies are external [\#843](https://github.com/nextcloud/nextcloud-dialogs/pull/843) ([@susnux](https://github.com/susnux))
+
+### Changed
+
+- Update Node engines to next LTS (version 20)
```
Comment (skjnldsv) [inline CHANGELOG.md:14]: Wouldn't it be breaking?

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (susnux) [reply CHANGELOG.md:14]: Theoretically yes, but it is not really breaking as it only emits a warning if the version mismatch (like some other dependencies).

But if you like we can go for 5.0

Comment (skjnldsv) [reply CHANGELOG.md:14]: Also depends on the content of the output.
No idea how far of a stretch this goes.
Would be nice to diff :thinking:

Comment (skjnldsv) [reply CHANGELOG.md:14]: I just checked, there is no diff between the two outputs.

```bash
19:35 [admin@workstation] ~/git/nextcloud-dialogs% md5sum dist/*  
5f31570fd715b54c648b3e68cafcd733  dist/filepicker.d.ts
3aea19086dad1679372420e989f0f0b6  dist/index.cjs
55798adc8c741ebcc24c2635cfc79602  dist/index.cjs.map
f1d8467142ed28a3b5b52276ba79d60a  dist/index.css
8a839f809c33ba7946f2e05837f0e7ec  dist/index.css.map
49a2d86aa8ef1730f88abc0d42dfdaf6  dist/index.d.ts
e89ed3615c74cc6a35ee20f94f0b20c0  dist/index.mjs
c2e9cb4aa56d4d1e32547f67e7fba2a1  dist/index.mjs.map
8bbc164800fbb83753fe95536cf33013  dist/l10n.d.ts
423857f8dc14bd68c8d51a43596fea9e  dist/toast.d.ts

19:35 [admin@workstation] ~/git/nextcloud-dialogs% md5sum dist-16/* 
5f31570fd715b54c648b3e68cafcd733  dist-16/filepicker.d.ts
3aea19086dad1679372420e989f0f0b6  dist-16/index.cjs
55798adc8c741ebcc24c2635cfc79602  dist-16/index.cjs.map
f1d8467142ed28a3b5b52276ba79d60a  dist-16/index.css
8a839f809c33ba7946f2e05837f0e7ec  dist-16/index.css.map
49a2d86aa8ef1730f88abc0d42dfdaf6  dist-16/index.d.ts
e89ed3615c74cc6a35ee20f94f0b20c0  dist-16/index.mjs
c2e9cb4aa56d4d1e32547f67e7fba2a1  dist-16/index.mjs.map
8bbc164800fbb83753fe95536cf33013  dist-16/l10n.d.ts
423857f8dc14bd68c8d51a43596fea9e  dist-16/toast.d.ts
```

**Comment by susnux** [comment]
@skjnldsv Could you create the GH release to trigger the npm workflow? (Repository admin permissions required)

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/nextcloud-dialogs/releases/tag/v4.1.0

**Comment by skjnldsv** [comment]
You're an admin now btw

**Pull request merged**